src/Link.js

```diff
@@ -27,8 +27,16 @@ class Link extends Component {
     this.onClick = this.onClick.bind(this);
   }
 
+  isAbsoluteURL(url) {
+    return url.startsWith("http://")
```

Comment:
Yes, but the problem is that I can't use the href property on the Button element, as it doesn't have it in most browsers. So instead I have to directly access the "href" attribute, which doesn't explode out relative paths.

So I am instead reintroducing that behaviour. I also don't have access to the `link.hostname` for comparison as I don't have access to the link object in the button, just a string.

Does that clarify things?